modules/api/pkg/handler/v2/routes_v2.go

```diff
@@ -1532,6 +1532,102 @@ func (r Routing) RegisterV2(mux *mux.Router, oidcKubeConfEndpoint bool, oidcCfg
 		Handler(r.getSeedOverview())
 }
 
+// swagger:route GET /projects/{project_id}/providers/aws/sizes project listProjectAWSSizes
```

Comment:
This (and the other endpoints below) is missing the `/api/v2/` prefix and causes the verify job to fail.